The C23 patch we added to media-libs/id3lib for https://bugs.gentoo.org/949086
means that 'bool' isn't exposed by the id3lib headers anymore (see
commit 5b0d1e373dcd8c4f79735d033d205a6f36698c59 for the details on why)
so we need to use 'my_bool' which *is* exposed by them instead when
apparently making up our own interfaces here in easytag.

See https://bugs.gentoo.org/949814.
--- a/src/tags/id3lib/id3_bugfix.h
+++ b/src/tags/id3lib/id3_bugfix.h
@@ -29,13 +29,13 @@
 G_BEGIN_DECLS
 
 #if !HAVE_DECL_ID3FIELD_SETENCODING
-ID3_C_EXPORT bool                  CCONV ID3Field_SetEncoding    (ID3Field *field, ID3_TextEnc enc);
+ID3_C_EXPORT my_bool                  CCONV ID3Field_SetEncoding    (ID3Field *field, ID3_TextEnc enc);
 #endif /* !HAVE_DECL_ID3FIELD_SETENCODING */
 #if !HAVE_DECL_ID3FIELD_GETENCODING
 ID3_C_EXPORT ID3_TextEnc           CCONV ID3Field_GetEncoding    (const ID3Field *field);
 #endif /* !HAVE_DECL_ID3FIELD_GETENCODING */
 #if !HAVE_DECL_ID3FIELD_ISENCODABLE
-ID3_C_EXPORT bool                  CCONV ID3Field_IsEncodable    (const ID3Field *field);
+ID3_C_EXPORT my_bool                  CCONV ID3Field_IsEncodable    (const ID3Field *field);
 #endif /* !HAVE_DECL_ID3FIELD_ISENCODABLE */
 ID3_C_EXPORT ID3_FieldType         CCONV ID3Field_GetType        (const ID3Field *field);
 //ID3_C_EXPORT ID3_FieldID           CCONV ID3Field_GetID          (const ID3Field *field);
diff --git a/src/tags/id3lib/c_wrapper.cpp b/src/tags/id3lib/c_wrapper.cpp
index ace7442..d7ad422 100644
--- a/src/tags/id3lib/c_wrapper.cpp
+++ b/src/tags/id3lib/c_wrapper.cpp
@@ -39,7 +39,7 @@ extern "C"
   // Tag wrappers
   //
 
-  ID3_C_EXPORT bool CCONV
+  ID3_C_EXPORT my_bool CCONV
   ID3Field_SetEncoding(ID3Field *field, ID3_TextEnc enc)
   {
     bool changed = false;
@@ -61,7 +61,7 @@ extern "C"
     return enc;
   }
 
-  ID3_C_EXPORT bool CCONV
+  ID3_C_EXPORT my_bool CCONV
   ID3Field_IsEncodable(const ID3Field *field)
   {
     bool isEncodable = false;
@@ -116,7 +116,7 @@ extern "C"
   // Call with :
   //    Mp3_Headerinfo* headerInfo = malloc(sizeof(Mp3_Headerinfo));
   //    ID3Tag_GetMp3HeaderInfo(tag, headerInfo);
-  /*ID3_C_EXPORT bool CCONV
+  /*ID3_C_EXPORT my_bool CCONV
   ID3Tag_GetMp3HeaderInfo(ID3Tag *tag, Mp3_Headerinfo* headerInfo)
   {
     const Mp3_Headerinfo* rem_headerInfo = NULL;
